docs/proposals/distributedstorage/distributedstorage.md

```diff
@@ -0,0 +1,594 @@
+
+**Feature**:  With the enhanced Kurator, Operators can easily configure distributed storage policies for multiple clusters simultaneously.
+
+**Value**: Provides a simplified, automated way to unify the management of distributed storage across multiple clusters. Reduces human error and ensures data continuity and compliance.
+
+**Outcome**: Using this feature, Operator can easily configure distributed storage for all clusters to improve the reliability, availability and storage efficiency of business system storage, as well as easy scalability
+
+##### Story 2
+
+**User Role**: Operator managing many different service which need different storage type
+
+**Feature**: Operator have the ability to slap different labels on the nodes where different services reside in Kubernetes. With the enhanced Kurator, based on these labels, Operator are able to specify the appropriate storage type for the corresponding service easily.
+
+**Outcome**: Using this feature, Operators can provide different types of storage for different services in all clusters to improve business system storage efficiency and overall service performance.
+
+#### Notes/Constraints/Caveats (Optional)
+
+<!--
+What are the caveats to the proposal?
+What are some important details that didn't come across above?
+Go in to as much detail as necessary here.
+This might be a good place to talk about core concepts and how they relate.
+-->
+
+- **Resource Require** In order to configure the Ceph storage cluster, at least one of these local storage options are required:
+    - Raw devices (no partitions or formatted filesystems)
+    - Raw partitions (no formatted filesystem)
+    - LVM Logical Volumes (no formatted filesystem)
+    - Persistent Volumes available from a storage class in block mode
+
+#### Risks and Mitigations
+
+<!--
+What are the risks of this proposal, and how do we mitigate? 
+
+How will security be reviewed, and by whom?
+
+How will UX be reviewed, and by whom?
+
+Consider including folks who also work outside the SIG or subproject.
+-->
+
+- **Version Compatibility** It is recommended that users use a newer version of Kubernetes, as older versions are not tested with Rook. Kubernetes v1.19 or higher is supported by Rook.
+
+### Design Details
+
+<!--
+This section should contain enough information that the specifics of your
+change are understandable. This may include API specs (though not always
+required) or even code snippets. If there's any ambiguity about HOW your
+proposal will be implemented, this is the place to discuss them.
+-->
+
+In this section, we will dive into the detailed API designs for the Unified Distributed Cloud Storage feature.
+
+These APIs are designed to facilitate Kurator's integration with Rook to enable the required functionality.
+
+In contrast to Rook, we may need to adapt the Unified Distributed Cloud Storage to reflect our new strategy and decisions.
+
+#### Unified Distributed Storage System API
```

Comment:
I think we should support fleet level storage. I was thinking about separating each plugin out from the fleet API. But even though, we should make them support fleet. And the implementation can be put into fleet manager